rero_ils/modules/documents/listener.py

```diff
@@ -102,7 +102,7 @@ def enrich_document_data(sender, json=None, record=None, index=None,
                 #   'nested' structure.
                 acq_date = item.get('acquisition_date')
                 if acq_date:
```

Comment:
Could be written in one line: `if item.get('acquisition_date'):`

Reply:
It's because we use `acq_date` two times : One for test, one for assignment (L109).
Using this method, we skip one dictionary search